server/game/gamesteps/challenge/challengeflow.js

```diff
@@ -164,9 +182,29 @@ class ChallengeFlow extends BaseStep {
             defenders = defenders.concat(this.forcedDefenders);
         }
 
-        this.challenge.addDefenders(defenders);
+        this.defendersToKneel = [];
```

Comment:
No need to make this an instance variable since it's set, used and reset in the same method. A local variable would be sufficient.